src/fragments/lib-v1/restapi/flutter/getting-started/40_postTodo.mdx

```diff
@@ -1,20 +1,48 @@
-```dart
-import 'dart:typed_data';
+<BlockSwitcher>
 
-Future<void> onTestApi() async {
-  try {
-    final options = RestOptions(
-        path: '/todo',
-        body: Uint8List.fromList('{\'name\':\'Mow the lawn\'}'.codeUnits)
-    );
-    final restOperation = Amplify.API.post(
-        restOptions: options
-    );
-    final response = await restOperation.response;
-    print('POST call succeeded');
-    print(String.fromCharCodes(response.data));
-  } on ApiException catch (e) {
-    print('POST call failed: $e');
+<Block name="Stable (Mobile)">
+
+  ```dart
+  import 'dart:typed_data';
+
+  Future<void> onTestApi() async {
+    try {
+      final options = RestOptions(
+          path: '/todo',
+          body: Uint8List.fromList('{\'name\':\'Mow the lawn\'}'.codeUnits)
+      );
+      final restOperation = Amplify.API.post(
+          restOptions: options
+      );
+      final response = await restOperation.response;
+      print('POST call succeeded');
+      print(response.body);
+    } on ApiException catch (e) {
+      print('POST call failed: $e');
+    }
   }
-}
-```
+  ```
+
+</Block>
+
+<Block name = "Developer Preview (Mobile, Web & Desktop)">
+
+  ```dart
+  Future<void> onTestApi() async {
```

Comment:
nit: I know this was the name before, but the other examples are all relevant to the operation. Could rename to `postTodo()`

Reply:
good suggestion, just updated